src/ontology/mondo-edit.obo

```diff
@@ -520167,6 +520139,52 @@ is_a: MONDO:0003847 {source="https://orcid.org/0000-0001-5208-3432"} ! hereditar
 relationship: has_characteristic MONDO:0021152 {source="OMIMPS:616033"} ! inherited
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/6777" xsd:anyURI
 
+[Term]
+id: MONDO:0800451
+name: congenital amegakaryocytic thrombocytopenia
+subset: rare
+synonym: "congenital amegakaryocytic thrombocytopenia" EXACT CLINGEN_PREFERRED [OMIMPS:604498]
+xref: OMIMPS:604498 {source="MONDO:equivalentTo"}
+is_a: MONDO:0100241 {source="DOID:0090118", source="MESH:C535982", source="MONDO:Redundant"} ! inherited thrombocytopenia
+relationship: excluded_subClassOf MONDO:0001713 {source="Orphanet:3319", source="https://orcid.org/0000-0001-5208-3432"} ! inherited aplastic anemia
+relationship: has_characteristic MONDO:0021152 {source="OMIMPS:604498"} ! inherited
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/6622" xsd:anyURI
+property_value: seeAlso "https://rarediseases.info.nih.gov/diseases/640/congenital-amegakaryocytic-thrombocytopenia" xsd:anyURI {source="GARD:0000640"}
+
+[Term]
+id: MONDO:0800452
+name: congenital amegakaryocytic thrombocytopenia 1
+def: "A rare inherited bone marrow failure syndrome, in which the cause of the disease is a variation in the MPL gene. It is characterized by an isolated and severe decrease in the number of platelets and megakaryocytes during the first years of life that develops into bone marrow failure with pancytopenia later in childhood." [https://orcid.org/0000-0001-5208-3432, Orphanet:3319]
+subset: gard_rare {source="GARD:640"}
+subset: nord_rare {source="MONDO:NORD"}
+subset: ordo_disease {source="Orphanet:3319"}
+subset: orphanet_rare {source="Orphanet:3319"}
+subset: rare
+synonym: "amegakaryocytic thrombocytopenia, congenital" EXACT [MONDO:Lexical, OMIM:604498]
+synonym: "CAMT1" EXACT ABBREVIATION [DOID:0090118, MONDO:Lexical, OMIM:604498, Orphanet:3319]
+synonym: "congenital amegakaryocytic thrombocytopenic purpura" EXACT [DOID:0090118, Orphanet:3319]
```

Comment:
[Orphanet:3319](https://www.orpha.net/consor/cgi-bin/Disease_Search.php?lng=EN&data_id=3053&Disease_Disease_Search_diseaseGroup=Congenital-amegakaryocytic-thrombocytopenia) and [DOID:0090118](https://disease-ontology.org/?id=DOID:0090118) are "Congenital amegakaryocytic thrombocytopenia" so this seems like it should also be a synonym for MONDO:0800451

Reply:
they both xref OMIM:604498, so I think it is appropriate to leave them here.

This is a bit subjective though, because OMIM probably created the OMIMPS and Orphanet and DO have not updated their mappings. If you feel strongly about moving them to the parent, I can do so.